master/reference/advanced/etcd-rbac/certificate-generation.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Generating certificates with usernames for etcd RBAC
```

Comment:
How about "Generating Certificates for etcd RBAC" because the guide also helps with the CA and and etcd certificates which do not include usernames?